target/linux/ar71xx/files/arch/mips/ath79/Kconfig.openwrt

```diff
@@ -1060,7 +1069,7 @@ config ATH79_MACH_WNR2000_V3
 	select ATH79_DEV_LEDS_GPIO
 	select ATH79_DEV_M25P80
 
-	config ATH79_MACH_WNR2200
```

Comment:
Please, don't include that kind of whitespace fixes withing your patch as it's not related at all.
If you want to fix this, do it in a separate commit.